vic/drivers/shared_all/src/calc_root_fraction.c

```diff
@@ -124,6 +134,7 @@ calc_root_fractions(veg_con_struct  *veg_con,
                 }
             }
             else if (Zsum + Zstep > Lsum) {
+                zone++;
```

Comment:
Looks good.  I'm assuming you've tested this to ensure that the root fractions come out the same as before?

I wonder if there's a simpler algorithm for this function.  Something to look into some day.